Source/Libraries/CorruptCore/CorruptCore.cs

```diff
@@ -242,7 +241,6 @@ public static void Start()
 
         public static void Shutdown()
         {
```

Comment:
This is used by one of the incomplete plugins. Just flag this as justified because it's an api call and at some point we can direct the api calls all through classes that make their intention clear.

Reply:
You're right, I believe we discussed this when I made the member private. I've restored `CorruptCoreExiting` and ignored the compiler warning.